src/containers/Secrets/Secrets.test.js

```diff
@@ -136,21 +139,25 @@ it('click add new secret & modal appears', () => {
 
   jest.spyOn(API, 'getServiceAccounts').mockImplementation(() => []);
 
-  const { getByTestId, getByText } = renderWithIntl(
+  const { getByTestId, getByText } = renderWithRouter(
     <Provider store={store}>
-      <Secrets {...props} intl={intl} />
-    </Provider>
+      <Route
+        path="/secrets"
```

Comment:
`paths.secrets.all()` from router utils
similar for URL below, `urls.secrets.all()`